### DISCUSSION TITLE
static admin login page

### DISCUSSION THREAD

**Discussion by kpeu3u** [abydahana/aksara#335] [Q&A]
Hi, i am creating a new theme for my project using Aksara cms. My problem is that i can not find a way to create static login page for administration area instead of modal login.
I don't want to have login or register on frontend pages.

Can you sugest a way to acomplish that?

**Comment by abydahana** [comment]
There’s a class name to open a page inside popup modal, it’s named “—modal”. You can remove this class from the login link or replace it with “—xhr” instead.

If you would to disable the public login or registration, you can override the auth module to the “/modules” folder and make a redirection to another page under the construct.
